pudb/run.py

```diff
@@ -5,33 +5,44 @@ def main():
     import sys
 
     from optparse import OptionParser
+    # TODO: update cmd string
     parser = OptionParser(
-            usage="usage: %prog [options] SCRIPT-TO-RUN [SCRIPT-ARGUMENTS]")
+            usage="usage: %prog [options] [SCRIPT-TO-RUN|-m MODULE] [ARGUMENTS]")
```

Comment:
This isn't related to this PR, but I noticed that if you run `python -m pudb`, the prog is set to `__main__.py`.